### ISSUE TITLE
SPIKE: Creating a new Localpool

### ISSUE THREAD

**Issue by mkristian** [buzzn/core#1194]
@dottorer @scsirdx since the flow has two steps - see https://buzzn-team.atlassian.net/wiki/spaces/DEV/pages/1898233/Flow+Creating+a+new+local+group

* create minimal localpool
* create owner, either one:
  * person
  * organisation

since we once decided to go for create things piece by piece and provide feedback on missing things.

add a map of `violations` to localpool like `'owner' => 'is missing'` and then the UI can use the 'key' in this case `owner` and knows that the owner incorrect.

from the concept these are **allowed** validation errors and such objects can be stored int he DB. we still have the **not allowed** validation errors which produce errors on the backend (unless caught beforehand on the client) on save or create.

needed endpoints
* create localpool - POST /localpools
* create person owner - POST/localpools/:id/person_owner
* create organization owner - POST /localpools/:id/organization_owner
* assign person as owner - PUT /localpools/:id/organization_owner/:id
* assign organziation as owner - PUT /localpools/:id/person_owner/:id

further the localpool will have a nested object called `owner` which can be retrieve via includes on localpool.

further we need endpoints to give back all persons and all organizations for the current_user. the nested `users` and `organizations` objects from a localpool is not sufficient but we need a sum of those over all localpools. i.e. we need toplevel endpoints:
* /api/admin/persons
* /api/admin/organizations
of course these endpoints will restrict the result to sum of localpool.persons and localpool.organizations of the localpools the current_user can see.

in short
- [x] high level validation rules on localpool #1224 
- [x] top level endpoints for persons and organizations #1199 
- [ ] nested endpoints for the owner
